astropy/io/fits/tests/test_image.py

```diff
@@ -1208,8 +1208,8 @@ def test_comp_image_hcompression_1_invalid_data(self):
 
         pytest.raises(ValueError, fits.CompImageHDU,
                       np.zeros((2, 10, 10), dtype=np.float32), name='SCI',
-                      compressionType='HCOMPRESS_1', quantizeLevel=16,
-                      tileSize=[2, 10, 10])
+                      compression_type='HCOMPRESS_1', quantize_level=16,
+                      tile_size=[2, 10, 10])
```

Comment:
The `ignore_warnings` decorator can probably be removed then ?

Reply:
Good catch, thanks!